src/devtools/client/debugger/src/components/shared/Modal.js

```diff
@@ -8,7 +8,7 @@ import PropTypes from "prop-types";
 import React from "react";
 import type { Node as ReactNode } from "react";
 import classnames from "classnames";
-import Transition from "react-transition-group/Transition";
+import { Transition } from "react-transition-group";
```

Comment:
This switches us to using the node_module, which has a couple of advantages

- we no longer rely on building a vendors bundle, which will be annoying to maintain
- we now have a versioned module which we can bump if we want improvements